dags/openshift_nightlies/tasks/utils/final_dag_status.py

```diff
@@ -0,0 +1,24 @@
+from os import environ
+from openshift_nightlies.util import var_loader, executor, constants
+from openshift_nightlies.models.release import OpenshiftRelease
+from openshift_nightlies.models.dag_config import DagConfig
+
+
+from airflow.operators.python import PythonOperator
+
+
+def final_status(**kwargs):
+    for task_instance in kwargs['dag_run'].get_task_instances():
+        if task_instance.current_state() != 'success' and task_instance.task_id != kwargs['task_instance'].task_id:
+            raise Exception("Task {} failed. Failing this DAG run".format(task_instance.task_id))
```

Comment:
This is a bit nit picky here but instead of iterating over and raising an exception on the first task we see that failed, can the logic just grab all the failed tasks and then raise an exception if there are any? That way the log can log all the failed tasks in case more than 1 fails. 

i.e. 

```python
failed_tasks = [task_instance.task_id for task_instance in kwargs['dag_run'].get_task_instances() if task_instance.current_state == 'failed'] 

if len(failed_tasks) > 0:
  raise Exception(f" Tasks {failed_tasks} failed") 

```

```